core/drivers/crypto/caam/include/caam_desc_helper.h

```diff
@@ -234,8 +234,9 @@ static inline void dump_desc(uint32_t *desc)
 /*
  * FIFO Store from register src of length len
  */
-#define FIFO_ST(src, len)                                                      \
-	(CMD_FIFO_STORE_TYPE | FIFO_STORE_OUTPUT(src) | FIFO_STORE_LENGTH(len))
+#define FIFO_ST(cla, src, len)                                                 \
+	(CMD_FIFO_STORE_TYPE | CMD_CLASS(cla) | FIFO_STORE_OUTPUT(src) |       \
+	FIFO_STORE_LENGTH(len))
```

Comment:
mind indentation (for consistency with other macro definitions)

Reply:
Fixed